server/android/src/android.rs

```diff
@@ -84,11 +83,12 @@ pub mod android {
             ),
         };
 
-        // logging_init_with_appender(settings.logging.clone(), &AndroidLogger {});
         logging_init(
             settings.logging.clone(),
             Some(Box::new(|config| config.custom(AndroidLogger {}))),
         );
+        log_panics::init();
+        log::info!("omSupply server starting...");
```

Comment:
that was just to keep them together, right?

Reply:
yes - and the documentation said to have `log_panics` after the logging was initialised 🤷